### PR TITLE
Styles for image alignment (pypa/warehouse#4023)

### PR DESCRIPTION
**Pull request by mila**
This PR applies styling to classes generated from restructuredText, but which are stripped now.

Fixes in https://github.com/pypa/readme_renderer/issues/112 must be applied first and published to render proper HTML.

### CODE REVIEW

**Comment by mila** [comment]
localhost_project_remt_

File: warehouse/static/sass/blocks/_project-description.scss [hunk 1]
```diff
@@ -162,6 +162,34 @@
   > a  {
     margin-top: $spacing-unit / 2;
   }
+
+  img.align-left,
+  .figure.align-left,
```
Comment (nlhkabu) [inline warehouse/static/sass/blocks/_project-description.scss]: is this supposed to be `figure.align-left` not `.figure.align-left` ?

**Comment by nlhkabu** [comment]
Ok, thanks for the clarification :)

**Comment by nlhkabu** [comment]
:tada:

**Pull request merged**